src/KeycloakContext.ts

```diff
@@ -1,21 +1,20 @@
 import { AuthContextProvider } from './api'
+import Keycloak from 'keycloak-connect'
 
 export class KeycloakContext implements AuthContextProvider {
-  public readonly request: any
-  public readonly accessToken: any
-  public readonly authenticated: boolean
+  public readonly request: Keycloak.GrantedRequest
+  public readonly accessToken: Keycloak.Token | undefined
 
-  constructor ({ req }: { req: any }) {
+  constructor ({ req }: { req: Keycloak.GrantedRequest }) {
     this.request = req
     this.accessToken = (req && req.kauth && req.kauth.grant) ? req.kauth.grant.access_token : undefined
-    this.authenticated = this.accessToken && !this.accessToken.isExpired()
   }
 
   public isAuthenticated (): boolean {
-    return this.authenticated
+    return (this.accessToken && !this.accessToken.isExpired()) ? true : false 
```

Comment:
```suggestion
    return this.accessToken && !this.accessToken.isExpired() 
```

Reply:
Trust me, I wanted this but in this case the TypeScript compiler does not allow your suggestion because `this.accessToken && !this.accessToken.isExpired()` can possibly evaluate to undefined when this.accessToken is undefined. Alternative would be to change the function return type to `boolean | undefined`